packages/icons/vaadin-iconset.js

```diff
@@ -3,13 +3,11 @@
  * Copyright (c) 2015 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/icon/vaadin-iconset.js';
-import '@vaadin/icon/vaadin-icon.js';
```

Comment:
This is technically a breaking change - for example, I had to update visual tests to use a different import.

In fact, this line shoudn't be there: the `vaadin-iconset` does not require importing `vaadin-icon`.
Also, this is how Lumo iconset is organized. So let's consider changing it here for consistency.

Reply:
Done in #5343